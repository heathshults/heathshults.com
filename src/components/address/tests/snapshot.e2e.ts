import { snapIt } from '../../../test';

const component = 'address';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('correctly', '<hs-address>Address line one</hs-address>');

    snap(
      'the address heading',
      `<hs-address>
        <hs-address-heading>Address header</hs-address-heading>
        Address line one<br />
        Address line two<br />
        Postcode
      </hs-address>`
    );
  });
});
