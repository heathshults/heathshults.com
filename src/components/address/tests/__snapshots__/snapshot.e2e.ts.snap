// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`address renders correctly 1`] = `"<hs-address class=\\"hydrated\\"><address class=\\"hs-address\\">Address line one</address></hs-address>"`;

exports[`address renders the address heading 1`] = `
"<hs-address class=\\"hydrated\\"><address class=\\"hs-address\\"><hs-address-heading class=\\"hydrated\\"><span class=\\"hs-address__heading\\">Address header</span></hs-address-heading>
        Address line one<br>
        Address line two<br>
        Postcode
      </address></hs-address>"
`;
